app/classifier/drawing-tools/freehand-helpers.coffee

```diff
@@ -0,0 +1,8 @@
+createPathFromCoords = (coordsArray) ->
+  [firstCoord, otherCoords...] = coordsArray
+  path = "M #{firstCoord.x},#{firstCoord.y} "
+  path += "L #{x},#{y} " for {x, y} in otherCoords
```

Comment:
This looks legit, but I'm wondering if the data ever needs to differentiate between an open path or a closed path. i.e. M 0 0 L 1 0 L 0 1 Z instead of M 0 0 L 1 0 L 0 1

(Edit: I have no idea how to work these GitHub reviews properly; let's go old school comments for a while.)